src/WasmComponent.Sdk/WasmComponent.Sdk.csproj

```diff
@@ -56,28 +58,47 @@
-            <PrebuiltWasmToolsTarget Include="x86_64-macos" Rid="osx-x64" Ext=".tar.gz" />
+            <WasmToolsTarget Include="aarch64-linux" Rid="linux-arm64" Ext=".tar.gz" />
+            <WasmToolsTarget Include="aarch64-macos" Rid="osx-arm64" Ext=".tar.gz" />
+            <WasmToolsTarget Include="x86_64-linux" Rid="linux-x64" Ext=".tar.gz" />
+            <WasmToolsTarget Include="x86_64-macos" Rid="osx-x64" Ext=".tar.gz" />
             <!-- tar on non-Windows often cannot handle zip archives.  Until we have a solution, we canonly publish the packages from the Windows build. 
                  See https://github.com/bytecodealliance/componentize-dotnet/issues/41 -->
-            <PrebuiltWasmToolsTarget Include="x86_64-windows" Rid="win-x64" Ext=".zip" ExeExt=".exe" Condition="$([MSBuild]::IsOSPlatform('Windows'))" />
+            <WasmToolsTarget Include="x86_64-windows" Rid="win-x64" Ext=".zip" ExeExt=".exe" Condition="$([MSBuild]::IsOSPlatform('Windows'))" />
         </ItemGroup>
         <ItemGroup>
-            <PrebuiltWasmToolsOutputs Include="tools\%(PrebuiltWasmToolsTarget.Rid)\wasm-tools%(PrebuiltWasmToolsTarget.ExeExt)" />
+            <PrebuiltWasmToolsOutputs Include="tools\%(WasmToolsTarget.Rid)\wasm-tools%(WasmToolsTarget.ExeExt)" />
         </ItemGroup>
     </Target>
 
-    <Target Name="DownloadNativeToolingCore" Inputs="@(PrebuiltWasmToolsOutputs)" Outputs="@(PrebuiltWasmToolsOutputs)">
-        <DownloadFile SourceUrl="$(PrebuiltWasmToolsBaseUrl)-%(PrebuiltWasmToolsTarget.Identity)%(PrebuiltWasmToolsTarget.Ext)" DestinationFolder="tools\temp" DestinationFileName="%(PrebuiltWasmToolsTarget.Rid)%(PrebuiltWasmToolsTarget.Ext)" />
-        <MakeDir Directories="tools\%(PrebuiltWasmToolsTarget.Rid)" />
-        <Exec Command="tar -xf &quot;temp/%(PrebuiltWasmToolsTarget.Rid)%(PrebuiltWasmToolsTarget.Ext)&quot; -C %(PrebuiltWasmToolsTarget.Rid) --strip-components=1" WorkingDirectory="tools" />
+    <Target Name="GetWacDependencies">
+        <ItemGroup>
+            <WacTarget Include="aarch64-unknown-linux-musl" Rid="linux-arm64"/>
+            <WacTarget Include="aarch64-apple-darwin" Rid="osx-arm64"/>
+            <WacTarget Include="x86_64-unknown-linux-musl" Rid="linux-x64"/>
+            <WacTarget Include="x86_64-apple-darwin" Rid="osx-x64"/>
+            <WacTarget Include="x86_64-pc-windows-gnu" Rid="win-x64" ExeExt=".exe"/>
+        </ItemGroup>
+        <ItemGroup>
+            <WacOutputs Include="tools\%(WacTarget.Rid)\wac%(WacTarget.ExeExt)" />
+        </ItemGroup>
+    </Target>
+
+    <Target Name="DownloadWac" Inputs="@(WacOutputs)" Outputs="@(WacOutputs)">
+        <MakeDir Directories="tools\%(WacOutputs.Rid)" />
+        <DownloadFile SourceUrl="$(WacBaseUrl)-%(WacTarget.Identity)" DestinationFolder="tools\%(WacTarget.Rid)" DestinationFileName="wac%(WacTarget.ExeExt)" />
+        <Exec Command="chmod +x tools\%(WacTarget.Rid)\wac" Condition="$([MSBuild]::IsOSPlatform('Linux'))"/>
```

Comment:
This is a hack to pass CI.  Need the wac release process to package it in a Tar file to make it executable